code/modules/mining/machinery/material_smelter.dm

```diff
@@ -38,7 +38,7 @@
 	var/adjusted_air = FALSE
 	for(var/mtype in reagents?.reagent_volumes)
 		var/decl/material/mat = GET_DECL(mtype)
-		if(mat.boiling_point && temperature >= mat.boiling_point)
+		if(!isnull(mat.boiling_point) && temperature >= mat.boiling_point)
```

Comment:
zero kelvins is still an invalid boiling point so i don't know if this is a necessary change